### PR TITLE
uses lowercase values for aggs and display 

### PR DESCRIPTION
**Pull request by Danwhy**
ref #515

### CODE REVIEW

File: lib/transforms/json-to-html-data.js [hunk 1]
```diff
@@ -325,13 +325,10 @@ function getObjectType (data, links) {
   var value;
   if (data.type !== 'people' && data.attributes.name) {
     value = data.attributes.name.map((el, i, arr) => {
-      var val = el.value;
```
Comment (SimonLab) [inline lib/transforms/json-to-html-data.js]: We don't want to add a comma to separate the values?

Comment (Danwhy) [inline lib/transforms/json-to-html-data.js]: no, the comma is added on the front end

Comment (SimonLab) [inline lib/transforms/json-to-html-data.js]: For all the values? I add to keep a similar code for separate the occupations for a person

Comment (Danwhy) [inline lib/transforms/json-to-html-data.js]: All of the values in the details block at the bottom, there's a helper that does it. I didn't apply it to the values at the top of the page

**Comment by nelsonic** [comment]
Yeah agree. But we can merge. 👍

**Pull request merged**